src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResponseTest.kt

```diff
@@ -52,7 +54,7 @@ internal class VisitResponseTest {
   ).apply {
     this.visitors.add(
       VisitVisitor(
```

Comment:
since person/address/phone is so easy - just dumped builders and went for raw objects